node/src/assets.rs

```diff
@@ -347,6 +348,32 @@ where
     last_id: Mutex<Option<UniqueId>>,
 }
 
+/// Iterates over a key range in column `db_col`, determined by  [`DistributedAssetStorage::<T>::make_prefix_range(my_participant_id, domain_id)`],
+/// Deletes all entries in `db_col` that evaluate `false` for `is_subset_of_active_participants(persistent_participants)`
+pub fn clean_db<T>(
+    db: &Arc<SecretDB>,
+    db_col: DBCol,
+    persistent_participants: &Vec<ParticipantId>,
```

Comment:
It's idiomatic to take references to slices instead of vecs

```suggestion
    persistent_participants: &[ParticipantId],
```